linker/Tests/Mono.Linker.Tests.Cases/Attributes/NoSecurity/CoreLibrarySecurityAttributeTypesAreRemoved.cs

```diff
@@ -28,6 +28,10 @@ namespace Mono.Linker.Tests.Cases.Attributes.NoSecurity {
 	// Fails with `Runtime critical type System.Reflection.CustomAttributeData not found` which is a known short coming
 	[SkipPeVerify (SkipPeVerifyForToolchian.Pedump)]
 	[SkipPeVerify ("System.dll")]
+
+	// System.Core is being preserved in the class libraries via reflection
+	// PeVerify fails on the GAC System.Core used on windows
```

Comment:
Can you explain in these comments that System.Core is being pulled in because System.dll references it, at least when linking against .net framework